README.md

```diff
@@ -0,0 +1,35 @@
+README:
+
+Team mike's UML Editor
```

Comment:
We can use up to six "#" to make a heading out of a line of text. Since this is the title of the whole thing, I say we go with four, which will end up looking like:

#### Team mike's UML editor